### PR TITLE
home page

### PR DESCRIPTION
**Pull request by prakashsakari**
This PR has the following.
1. Home page, that lists the video thumbnails and some information about the video done with API calls
2. Navbar
3. Sidebar with multiple options like playlist, watch later, history, liked video, and home.

Preview Link: https://deploy-preview-1--videoporium.netlify.app/

### CODE REVIEW

**Review by ranitasahaa1230 — COMMENTED** [review]
Nice and clean code, 
Good naming conventions followed
Good for approval

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    border: 1px solid var(--accent-color);
+}
+  
+.search-icon {
+    top: 8px;
+    left: 4px;
+}
+  
+.search-box-container {
+    margin-left: 0;
```
Comment (arpit01923) [inline src/components/Navbar/Navbar.css:35]: may use it in common.css file import it

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+  
+.navigation {
+    margin-left: 0;
```
Comment (arpit01923) [inline src/components/Navbar/Navbar.css:39]: same

File: src/components/Navbar/Navbar.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
+import { Link } from "react-router-dom";
+import logo from "../../assets/logo.png"
```
Comment (arpit01923) [inline src/components/Navbar/Navbar.jsx:2]: use absolute path for this to avoid ../../

File: src/pages/Home/Home.js [hunk 4]
```diff
@@ -0,0 +1,63 @@
+import { Navbar, Footer, SideBar, VideoCard, Category } from "../../components";
```
Comment (arpit01923) [inline src/pages/Home/Home.js:1]: same use absolute path here

**Review by arpit01923 — APPROVED** [review]
- clean UI
- good folder structure
- left some changes, work on it

File: src/components/Navbar/Navbar.css (hunk 1 above)
Comment (prakashsakari) [reply src/components/Navbar/Navbar.css:35]: ok will do this

File: src/components/Navbar/Navbar.css (hunk 2 above)
Comment (prakashsakari) [reply src/components/Navbar/Navbar.css:39]: thanks for notifying this

File: src/components/Navbar/Navbar.jsx (hunk 3 above)
Comment (prakashsakari) [reply src/components/Navbar/Navbar.jsx:2]: will read about this

**Pull request merged**